build.sh

```diff
@@ -67,19 +74,15 @@ pushd $WEBRTC_SRC || exit 1
 rm -rf out/$CONFIG
 popd
 
-echo "Applying webrtc patches ..."
-pushd $WEBRTC_SRC || exit 1
-for PATCH in build_at_webrtc_branch_heads_52.patch; do
-	git apply --check ${PROJECT_DIR}/webrtc_patches/${PATCH} || exit 1
-	git am < ${PROJECT_DIR}/webrtc_patches/${PATCH} || exit 1
-done
-popd
-
 echo "Building webrtc ..."
 pushd $WEBRTC_SRC
 export GYP_DEFINES="include_tests=0 include_examples=0"
-python webrtc/build/gyp_webrtc webrtc/api/api.gyp || exit 1
-ninja -C out/$CONFIG || exit 1
+if [ "$ARCH" = "arm" ]; then
+    gn gen out/$CONFIG --args='target_os="linux" target_cpu="arm" is_debug=false' || exit 1
+else
+    gn gen out/$CONFIG --args='is_debug=false' || exit 1
+fi
+ninja -C out/$CONFIG webrtc webrtc/test webrtc/pc:pc_test_utils || exit 1
```

Comment:
I think `ninja -C out/$CONFIG webrtc field_trial metrics_default fake_audio_device` is sufficient.